packages/vm/README.md

```diff
@@ -218,17 +236,17 @@ If you want to understand your VM runs we have added a hierarchically structured
 
 The following loggers are currently available:
 
-| Logger | Description |
-| - | - |
-| `vm:block` | Block operations (run txs, generating receipts, block rewards,...) |
-| `vm:tx` | Transaction operations (account updates, checkpointing,...) |
-| `vm:tx:gas` | Transaction gas logger |
-| `vm:evm` | EVM control flow, CALL or CREATE message execution |
-| `vm:evm:gas` | EVM gas logger |
-| `vm:eei:gas` | EEI gas logger |
-| `vm:state`| StateManager logger |
-| `vm:ops` | Opcode traces |
-| `vm:ops:[Lower-case opcode name]` | Traces on a specific opcode |
+| Logger                            | Description                                                        |
+| --------------------------------- | ------------------------------------------------------------------ |
```

Comment:
Same as the comment above, should I remove these non-related changes?

Reply:
This one is kind of annoying, I like the prettier format in the code sections especially, but could do without these table formatting. I wonder if we could turn off table formatting specifically in our prettier root config file.